src/routes/index.js

```diff
@@ -7,7 +7,7 @@ import Quiz from '../components/Quiz';
 export default () => (
   <Switch>
     <Route exact path="/" component={Home} />
-    <Route path="/quiz" component={Quiz} />
+    <Route path="/:quizName" component={Quiz} />
```

Comment:
This line needs to be below the `/about` line. When it is here, if I go to `/about` then the app throws an error because it thinks `about` is a `quizName`.

Also, in an other issue/pull request we can figure out what to do if a user goes to a non-existent path like `/python`

Reply:
Changed